plugin.cpp

```diff
@@ -1,3 +1,9 @@
+/*
+ * Copyright (C) 2016, Pelagicore AB.
+ * Author: Jonatan Pålsson <jonatan.palsson@pelagicore.com>
+ * SPDX-License-Identify: MPL-2.0
+ */
```

Comment:
@jonte The templates are there to be the "current truth" so to speak. I.e. they enable us to not have the same discussions every time we create a new code file, but they are also intended to be improved over time when anyone have good input.